package-lock.json

```diff
@@ -2426,6 +2426,30 @@
         "vue-router": "^4.4.5"
       }
     },
+    "node_modules/@kong-ui-public/core": {
+      "version": "1.10.6",
+      "resolved": "https://registry.npmjs.org/@kong-ui-public/core/-/core-1.10.6.tgz",
+      "integrity": "sha512-ksi4HTCKdhrpUldvBQjYv5iDEphHuZ6at+S6VTaD/sEbPO4X2+FgtZ/la3MJuYiRvvjkgtPWNL+jNh0ScH8SYg==",
+      "license": "Apache-2.0",
+      "dependencies": {
+        "date-fns": "^4.1.0"
+      },
+      "peerDependencies": {
+        "axios": "^1.7.7",
+        "swrv": "^1.0.4",
+        "vue": "^3.5.12"
+      }
+    },
+    "node_modules/@kong-ui-public/core/node_modules/date-fns": {
+      "version": "4.1.0",
+      "resolved": "https://registry.npmjs.org/date-fns/-/date-fns-4.1.0.tgz",
+      "integrity": "sha512-Ukq0owbQXxa/U3EGtsdVBkR1w7KOQ5gIBqdH2hkvknzZPYvBxb/aa6E8L7tmjFtkwZBu3UXBbjIgPo/Ez4xaNg==",
+      "license": "MIT",
+      "funding": {
+        "type": "github",
+        "url": "https://github.com/sponsors/kossnocorp"
+      }
+    },
```

Comment:
Doesn't block approval, but more to note for me and @schogges 

---

"Uh-oh" 😄